target/linux/generic/pending-5.4/483-mtd-spi-nor-add-support-for-Boyamicro-BY25Q128AS.patch

```diff
@@ -0,0 +1,26 @@
+From: George Brooke <figgyc@figgyc.uk>
+Subject: [PATCH] mtd: spi-nor: add support for Boya BY25Q128AS
+
+Adds support for Boya Microelectronics (BoyaMicro) BY25Q128AS 128 MiB flash.
+Tested on Creality WB-01 embedded device which uses this chip.
```

Comment:
Is this submitted upstream, i.e. correctly placed in the pending-* directory?

Reply:
Not currently, apologies. Would you rather I move it to the correct folder or submit it to upstream? (also working on the other changes, just need to do some testing for those so it'll be a bit)

Reply:
Submitting upstream is obviously the desired option, as this will have the proper authority care about the change.